Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1493,6 +1494,9 @@ eval (UpdateLocAndLatLong lat lng) state = do
     pure NoAction
   ]
 
+eval GoToMyProfile_ state = do
```

Comment:
@aryanguptaSG can we use some other name for this action which is more intuitive. Because this action is triggered when the user wants to edit the profile.